src/main/java/org/terasology/structureTemplates/internal/systems/StructureSpawnServerSystem.java

```diff
@@ -186,10 +196,11 @@ public void onGetStructureTemplateBlocks(GetStructureTemplateBlocksEvent event,
             }
         }
 
+
         List<BuildStep> blocksPerStep = Lists.newArrayList(blocksPerLayer.values()).stream().map(BuildStep::new).collect(Collectors.toList());
         BuildStepwiseStructureComponent buildStepwiseStructureComponent = new BuildStepwiseStructureComponent(blocksPerStep);
         BuildStructureCounterComponent growStructureCounter = new BuildStructureCounterComponent();
-
+        location = blockRegionList.get(0).region.getMin(new Vector3i());
```

Comment:
So we just take the minimum corner of the first block region in the template as the location at which we place the multi-block root?

I'm wondering whether using the origin of the ST would be better (when you create STs in-game you start the selection from somewhere, and that block will be the origin of the local coordinate system iirc).

Reply:
I can't think of any added benefit of having the origin as the main location as the main location of the multiBlock is useful only to locate the multiBlockEntity with the necessary information. I could give this a try though.

Reply:
Alright then, let's leave it as the first minimum corner we can find. No need to change this.